app/javascript/dashboard/routes/dashboard/settings/reports/components/Heatmap.vue

```diff
@@ -121,188 +147,39 @@ export default {
       return days[dayIndex];
     },
     getHeatmapLevelClass(value) {
-      if (!value) return '';
+      if (!value)
+        return 'outline-slate-100 dark:outline-slate-700 dark:bg-slate-700/40 bg-slate-50/50';
 
-      const level = [...this.quantileRange, Infinity].findIndex(
+      let level = [...this.quantileRange, Infinity].findIndex(
         range => value <= range && value > 0
       );
 
-      if (level > 6) {
-        return 'l6';
+      if (level > 6) level = 5;
+
+      if (level === 0) {
+        return 'outline-slate-100 dark:outline-slate-700 dark:bg-slate-700/40 bg-slate-50/50';
       }
 
-      return `l${level}`;
+      const classes = [
+        'bg-woot-50 dark:bg-woot-800/40 dark:outline-woot-800/80',
+        'bg-woot-100 dark:bg-woot-800/30 dark:outline-woot-800/80',
+        'bg-woot-200 dark:bg-woot-500/40 dark:outline-woot-700/80',
+        'bg-woot-300 dark:bg-woot-500/60 dark:outline-woot-600/80',
+        'bg-woot-600 dark:bg-woot-500/80 dark:outline-woot-500/80',
+        'bg-woot-800 dark:bg-woot-500 dark:outline-woot-400/80',
+      ];
+
+      return classes[level - 1];
     },
   },
 };
 </script>
 
 <style scoped lang="scss">
-$heatmap-colors: (
-  level-1: var(--w-50),
-  level-2: var(--w-100),
-  level-3: var(--w-300),
-  level-4: var(--w-500),
-  level-5: var(--w-700),
-  level-6: var(--w-900),
-);
-
-$heatmap-hover-border-color: (
-  level-1: var(--w-25),
-  level-2: var(--w-50),
-  level-3: var(--w-100),
-  level-4: var(--w-300),
-  level-5: var(--w-500),
-  level-6: var(--w-700),
-);
-
-$tile-height: 1.875rem;
-$tile-gap: var(--space-smaller);
-$container-gap-row: var(--space-one);
-$container-gap-column: var(--space-two);
-$marker-height: var(--space-two);
-
-@mixin heatmap-level($level) {
-  $color: map-get($heatmap-colors, 'level-#{$level}');
-  background-color: $color;
-  &:hover {
-    border: 1px solid map-get($heatmap-hover-border-color, 'level-#{$level}');
-  }
-}
-
-@media screen and (max-width: 768px) {
-  .heatmap-container {
-    overflow-y: auto;
-  }
-}
-
-.loading-cell {
-  background-color: var(--color-background-light);
-  border: 0px;
-
-  animation: loading-pulse 2s cubic-bezier(0.4, 0, 0.6, 1) infinite;
-}
-
-@keyframes loading-pulse {
-  0%,
-  100% {
-    opacity: 1;
-  }
-  50% {
-    opacity: 0;
-  }
-}
-
 .heatmap-container {
-  display: grid;
-  position: relative;
-  width: 100%;
-  gap: $container-gap-row $container-gap-column;
+  @apply grid relative w-full gap-x-4 gap-y-2.5 overflow-y-scroll md:overflow-visible;
   grid-template-columns: 80px 1fr;
```

Comment:
Can we move the `style="grid-template-columns: 80px 1fr"` to inline `grid-cols-[80px_1fr]`?